src/components/ScaleLattice.vue

```diff
@@ -0,0 +1,340 @@
+otonalFundamental
```

Comment:
What's this?

Reply:
No idea, it comes from your branch.

Reply:
Oops, my bad! :sweat_smile: Can be removed.